src/components/pill/pill.spec.tsx

```diff
@@ -588,6 +610,123 @@ describe("Pill", () => {
               });
             });
           });
+
+          describe("when pillRole is status and isDarkbackground prop is set to true", () => {
+            const pillRole = "status";
+            const styleSet = styleConfig(true)[pillRole];
+
+            describe.each(modernStyleTypes)(
+              'when the pill style is set as "%s"',
+              (style) => {
+                it("when storybook supplies the correct theme, matches the expected styling", () => {
```

Comment:
**suggestion**: using prop name in test description

To help with readability, I'd suggest using the name of prop here when describing the test's preconditions so its a bit clearer

```suggestion
              'when colorVariant prop is "%s"',
              (colorVariant) => {
                it("component renders with expected styling", () => {
```